apps/search-poc/src/containers/ResourceDetails/index.ts

```diff
@@ -1 +1,29 @@
-export { default } from './ResourceDetails';
\ No newline at end of file
+
+import React from 'react';
+
+
+const MorphViewer = React.lazy(() => import('./ReconstructedNeuronMorphology'));
+
+const compByName = {
+  morphViewer: MorphViewer,
+};
+
+const compNamesByType = {
```

Comment:
can we rename this to 
```suggestion
const componentsByType = {
```